config/cesm/config_grids.xml

```diff
@@ -120,6 +120,15 @@
       <grid name="rof">null</grid>
     </model_grid>
 
+    <!-- Regional NLDAS grid over the U.S. (roughly 12km resolution), with the standard NLDAS mask -->
```

Comment:
Suggestions: 

Regional NLDAS-2 grid over the U.S. (0.125 degree resolution; 25-53N, 235-293E), with the standard NLDAS-2 mask